meta-resin-common/recipes-core/images/resin-image.bb

```diff
@@ -25,6 +25,7 @@ IMAGE_FEATURES_append = " \
     ${@bb.utils.contains('DISTRO_FEATURES', 'development-image', 'debug-tweaks', '', d)} \
     splash \
     ssh-server-dropbear \
+    read-only-rootfs \
```

Comment:
Can you rebase your PR please and remove this current commit as well as also remove https://github.com/resin-os/meta-resin/pull/732/commits/55e6cb40a868a5c696c219714befc38cbbf5f648 please?